admin.js

```
@@ -3093,6 +3112,141 @@ class AdminPanel {
+  loadCabinManagerEmails(cabinManagerEmails) {
+    const listContainer = document.getElementById('cabinManagerEmailsList');
+
+    if (!listContainer) {
+      console.warn('Cabin manager emails list container not found');
+      return;
+    }
+
+    if (!cabinManagerEmails || cabinManagerEmails.length === 0) {
+      listContainer.innerHTML = `
+        <p style="color: #666; font-style: italic; padding: 1rem; background: var(--gray-50); border-radius: var(--radius-md);">
+          Žádní správci chaty nejsou zatím přidáni.
```

Biome (2.1.2)

[error] Duplicate class member name "escapeHtml"



(lint/suspicious/noDuplicateClassMembers)